public/jsx/components/card-form.jsx

```diff
@@ -69,54 +71,25 @@ module.exports = React.createClass({
     router: React.PropTypes.func,
   },
 
-  cardPatterns: {
-    'default': {
-      number: {
-        pattern: '1111 1111 1111 1111',
-        placeholder: gettext('Card number'),
-      },
-      cvv: {
-        pattern: '111',
-        placeholder: gettext('CVV'),
-      },
-      expirationDate: {
-        label: gettext('Expiry Date'),
-        pattern: '11/11',
-        placeholder: 'MM/YY',
-      },
-    },
-    'american-express': {
-      number: {
-        pattern: '1111 111111 11111',
-      },
-      cvv: {
-        pattern: '1111',
-        placeholder: gettext('CID'),
-      },
-    },
-    'diners-club': {
-      number: {
-        pattern: '1111 111111 1111',
-      },
-      cvv: {
-        pattern: '111',
-        placeholder: gettext('CVV'),
-      },
-    },
-  },
-
-
-  handleChange: function(index, e) {
-    var stateChange = {};
-    var val = e.target.value;
-    stateChange[e.target.id] = val;
-    this.setState(stateChange);
+  cardPatterns: cardPatterns,
```

Comment:
Yeah, fair point. I did this early on in trying to clean things up. I might see if it can be merged with the other form object instead.